frontend/app/src/utils/history.ts

```diff
@@ -14,9 +14,10 @@ import {
   TransactionEventType
 } from '@/types/transaction';
 
-export const getEventType = (
-  event: EthTransactionEventEntry
-): TransactionEventType | undefined => {
+export const getEventType = (event: {
+  eventType?: string | null;
```

Comment:
Can the type be any of the 3 states? undefined/string/null?

Reply:
yes, need to follow type from EthTransactionEvent 
```
eventType: z.string().nullish(),
eventSubtype: z.string().nullish(),
```

Reply:
Sure, but in the returned data we can have both missing (undefined) and null for the same field, and does missing have a different meaning than undefined?

Reply:
Yeah, it worked, since the optional operator (?) will accept undefined/missing. I believe undefined and missing is the same thing.